src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/ExcessHetUnitTest.java

```diff
@@ -171,15 +171,15 @@ public void testAllHetsForLargeCohorts() {
         singletonGTs.add(makeG("het0", Aref, T, hetPLs));
 
         final VariantContext singleton = makeVC("singleton", Arrays.asList(Aref, T), singletonGTs.toArray(new Genotype[singletonGTs.size()]));
-        final double singletonValue = new ExcessHet().calculateEH(singleton, singleton.getGenotypes()).getValue();
+        final double singletonValue = ExcessHet.calculateEH(singleton, singleton.getGenotypes()).getValue();
 
         final List<Genotype> allHetGTs = new ArrayList<>();
         for (int i = 0; i < numGTs; i++) {
             allHetGTs.add(makeG("het" + i, Aref, T, hetPLs));
         }
 
         final VariantContext allHet = makeVC("allHet", Arrays.asList(Aref, T), allHetGTs.toArray(new Genotype[allHetGTs.size()]));
-        final double hetsValue = new ExcessHet().calculateEH(allHet, allHet.getGenotypes()).getValue();
+        final double hetsValue = ExcessHet.calculateEH(allHet, allHet.getGenotypes()).getValue();
```

Comment:
Can you confirm that there is at least one test in `ExcessHetUnitTest` that fails if rounding were mistakenly turned **off** in the annotation class? If there isn't such a test, could you add one?